### ISSUE TITLE
Display in front-end when the error is that no DNS result is returned

### ISSUE THREAD

**Issue by gwire** [openrightsgroup/cmp-issues#118]
Example https://www.blocked.org.uk/results?url=http://emp3world.cc

This is a site that has a court order to be blocked, but I think the domain expired shortly after and was removed by the root server operators of the .cc root (Verisign).  Therefore a normal request would be expected to work, regardless of blocking.

(It's therefore confusing as to why BT-Light is showing "ok" for a recent check - are we not detecting DNS injection by BT for dead domains?)

Suggest "DNS error" in the front-end or similar.

**Comment by dantheta** [comment]
Ah - you've found something very interesting on the BT line that we didn't know was there!

It's an unresolveable domain search page.

It appears to be implemented using an http transparent proxy.  Direct DNS queries correctly return no results, but trying to make an HTTP request to http://doesnot.exist gives an HTML page with a redirect to the address http://www.webaddresshelp.bt.com.

I'm pretty sure this wasn't here before.  We had a similar problem with Virgin, and managed to turn off the "feature".

The full address that the site redirects to is:

http://www.webaddresshelp.bt.com/main?InterceptSource=0&ClientLocation=uk&ParticipantID=XXXXXXX&FailureMode=1&SearchQuery=&FailedURI=http%3A%2F%2Fdoesnot.exist%2F&AddInType=4&Version=2.1.8-1.90base&Referer=&Implementation=0&method=GET

I've blanked out the participant ID in the link above.

We can disable the function as we did for Virgin.  I'm fairly sure there was an issue somewhere for having the probe detect DNS search pages during startup.
